src/db.rs

```diff
@@ -1544,6 +1544,68 @@ impl<T: ThreadMode, D: DBInner> DBCommon<T, D> {
         }
     }
 
+    pub fn put_with_ts_opt<K, V, S>(
```

Comment:
Could you please add comment-docs to all public methods?

Reply:
done